From 1e14a6a8bf3133a4e10681635151ec463a8a1776 Mon Sep 17 00:00:00 2001
From: Moein Khazraee <mkhazrae@cs.ucsd.edu>
Date: Thu, 5 Aug 2021 08:35:03 -0700
Subject: [PATCH] Added SparSDR message to iio init.

---
 board/pluto/S23udc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/board/pluto/S23udc b/board/pluto/S23udc
index b599f699b6..f8cc17fcfc 100644
--- a/board/pluto/S23udc
+++ b/board/pluto/S23udc
@@ -56,6 +56,8 @@ create_iiod_context_attributes() {
 		cat /opt/${CALIBFILENAME} | grep ^cal,* >> /etc/libiio.ini
 		cat /opt/${CALIBFILENAME_TEMP_LUT} | grep ^cal,* >> /etc/libiio.ini
 	fi
+
+  echo "sparsdr_format_version=1" >> /etc/libiio.ini
 }
 
 case "$1" in
-- 
2.17.1

