From 21d6506dbe2f2aa257416978931c487fb7b48c46 Mon Sep 17 00:00:00 2001
From: Moein Khazraee <mkhazrae@cs.ucsd.edu>
Date: Thu, 5 Aug 2021 10:13:59 -0700
Subject: [PATCH] Updated SparSDR version number in iio init.

---
 board/pluto/S23udc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/pluto/S23udc b/board/pluto/S23udc
index f8cc17fcfc..f063baa257 100644
--- a/board/pluto/S23udc
+++ b/board/pluto/S23udc
@@ -57,7 +57,7 @@ create_iiod_context_attributes() {
 		cat /opt/${CALIBFILENAME_TEMP_LUT} | grep ^cal,* >> /etc/libiio.ini
 	fi
 
-  echo "sparsdr_format_version=1" >> /etc/libiio.ini
+  echo "sparsdr_format_version=2" >> /etc/libiio.ini
 }
 
 case "$1" in
-- 
2.17.1

